elements/urth-core-dataframe/urth-core-query-behavior.html

```diff
@@ -0,0 +1,40 @@
+<!--
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+-->
+
+<!--
+This behavior contains a set of utility methods that can be shared by
+query elements.
```

Comment:
Hmm... it's there further down https://github.com/jupyter-incubator/declarativewidgets/pull/354/files#diff-8e90bf9daa6d81bcdf11049e818f7a92R19

Is the issue that it has JS and HTML style comments? There are several behaviors like that. What is proper way?